server/server.go

```diff
@@ -171,3 +177,77 @@ func (server *HTTPServer) Start() error {
 func (server *HTTPServer) Stop(ctx context.Context) error {
 	return server.Serv.Shutdown(ctx)
 }
+
+// redirectTo
+func (server HTTPServer) redirectTo(baseURL string) func(http.ResponseWriter, *http.Request) {
+	return func(writer http.ResponseWriter, request *http.Request) {
+		endpointURL, err := server.composeEndpoint(baseURL, request.RequestURI)
+
+		if err != nil {
+			log.Error().Err(err).Msg("Error during endpoint URL parsing")
+			handleServerError(writer, err)
+		}
+
+		// test service available
+		_, err = http.Get(endpointURL.String())
+		if err != nil {
+			log.Error().Err(err).Msg("Aggregator service unavailable")
+
+			if _, ok := err.(*url.Error); ok {
+				err = &AggregatorServiceUnavailableError{}
+			}
+
+			handleServerError(writer, err)
+		}
+
+		log.Info().Msgf("Redirecting to %s", endpointURL.String())
+		http.Redirect(writer, request, endpointURL.String(), 302)
+	}
+}
+
+func (server HTTPServer) proxyTo(baseURL string) func(http.ResponseWriter, *http.Request) {
+	return func(writer http.ResponseWriter, request *http.Request) {
+		log.Info().Msg("Handling response as a proxy")
+		endpointURL, err := server.composeEndpoint(baseURL, request.RequestURI)
+
+		if err != nil {
+			log.Error().Err(err).Msgf("Error during endpoint %s URL parsing", request.RequestURI)
+			handleServerError(writer, err)
+		}
+
+		client := http.Client{}
+		req, _ := http.NewRequest(request.Method, endpointURL.String(), request.Body)
+		copyHeader(request.Header, req.Header)
+
+		log.Debug().Msgf("Connecting to %s", endpointURL.String())
+		response, err := client.Do(req)
+		if err != nil {
+			log.Error().Err(err).Msgf("Error during retrieve of %s", endpointURL.String())
+			handleServerError(writer, err)
+		}
+
+		content, err := ioutil.ReadAll(response.Body)
+
+		if err != nil {
+			log.Error().Err(err).Msgf("Error while retrieving content from request to %s", endpointURL.String())
+			handleServerError(writer, err)
+		}
+		// Maybe this code should be on responses.SendRaw or something like that
+		writer.Header().Set("content-type", "application/json; charset=utf-8")
+		writer.WriteHeader(http.StatusOK)
+		writer.Write(content)
```

Comment:
If PR [#56 of Operator Utils](https://github.com/RedHatInsights/insights-operator-utils/pull/56) is accepted and merged, this code should be refactored to a simply `responses.Send(http.StatusOK, writer, content)`